.eslintrc

```diff
@@ -25,9 +25,13 @@
     "plugin:react/recommended",
     "standard",
     "standard-jsx",
-    "standard-react"
+    "standard-react",
+    "plugin:import/errors",
+    "plugin:import/warnings"
   ],
   "rules": {
+    "import/no-unresolved": 0, 
```

Comment:
@louh this will add non-blocking warning about import orders, which I used for refactoring. After we merge, we could probably run a separate PR running this as a `--fix` to tidy things up